src/azure-cli/requirements.py3.Darwin.txt

```diff
@@ -118,9 +118,10 @@ PyGithub==1.55
 PyJWT==2.4.0
 PyNaCl==1.5.0
 pyOpenSSL==25.0.0
+PySocks==1.7.1
```

Comment:
Do we directly rely on `PySocks`? What is the problem if `PySocks==1.7.1` is installed via `requests[socks]`?

Reply:
See https://github.com/Azure/azure-cli/pull/31708/files#r2165880791.